server/src/main/java/io/druid/client/HttpServerInventoryViewConfig.java

```diff
@@ -24,25 +24,39 @@
 import com.google.common.base.Preconditions;
 import org.joda.time.Period;
 
+import java.util.concurrent.TimeUnit;
+
 /**
  */
 public class HttpServerInventoryViewConfig
 {
+  // HTTP request timeout
   @JsonProperty
   private final long serverTimeout;
 
+  // Requests to server may fail when it is shutsdown abruptly and there is a lag in coordinator
+  // discovering its disappearance. So, failure would be logged only after the acceptable
+  // unstableTimeout has passed.
+  @JsonProperty
+  private final long serverUnstabilityTimeout;
+
   @JsonProperty
   private final int numThreads;
 
   @JsonCreator
   public HttpServerInventoryViewConfig(
       @JsonProperty("serverTimeout") Period serverTimeout,
+      @JsonProperty("serverUnstabilityTimeout") Period serverUnstabilityTimeout,
```

Comment:
this object is only ever deserialized from config, it would never be serialized and then deserialized back so different types are ok in this context.